pkgs/applications/misc/grsync/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, dee, gtk2, intltool, libdbusmenu-gtk2, libunity, pkg-config, rsync }:
+
+stdenv.mkDerivation rec {
+  version = "1.2.6";
+  pname = "grsync";
+
+  src = fetchurl {
+    url = "https://kent.dl.sourceforge.net/project/grsync/grsync-${version}.tar.gz";
```

Please use the mirror url
```suggestion
    url = "mirror://sourceforge/grsync/grsync-${version}.tar.gz";
```
besides that, LGTM

---

Thanks